### ISSUE TITLE
Sympa Accessibility Issue - 15 - No clear visual focus indicator when accessing elements by keyboard.

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1744]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
Using a Screen Reader, there needs to be a clear visual focus indicator when access elements by keyboard.

Actual behavior
---------------
There is no clear visual focus indicator when accessing elements by keyboard. Users with poor vision would not understand where the current focus is. In the screenshot below, my current focus is on word prixlist, but since it doesn’t have a focused box/border around the element, it is not distinguishable.

[WCAG Level AA 2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html))

![image](https://github.com/sympa-community/sympa/assets/144282683/d72a1d0d-1f44-45ea-b207-c894cced4432)

Fig 17 Focus indictor issue.

Steps to reproduce
------------------
Sympa Accessibility Issue -15 was determined by manual functional testing performed by the user navigating with keyboard only or using assistive technology such as screen reader.   See attached review for more info on specific location in interface.

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13494310/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)

**Comment by ldidry** [comment]
I quickly tried some CSS rules:
```css
#sympa-top-bar menu .menu li:not(.active) > a:not(.button):focus {
    background: #6A5687 !important;
}
a:focus {
    border-bottom-style: solid;
    border-bottom-width: 1px;
}
```

@ikedas @racke, could you test them and tell what do you think of them?

Same question for @rm1-uiuc if you can test them with a Sympa.

**Comment by ikedas** [comment]
I have a question: What style should we adopt for "focus indicator"?

* [G149](https://www.w3.org/WAI/WCAG21/Techniques/general/G149) and so on provide some examples.
* [G165](https://www.w3.org/WAI/WCAG21/Techniques/general/G165) suggests that *default focus indicator for the platform* should be used.
* [G195](https://www.w3.org/WAI/WCAG21/Techniques/general/G195) suggests that *author-supplied, visible focus indicator* should be used.

**Comment by ikedas** [comment]
I realized that removing these lines recover the default focus indicator:

https://github.com/sympa-community/sympa/blob/4f62c3a6c4d0156d02273f49ab7fd16e9d0d371a/default/web_tt2/css.tt2#L202-L205

With Chrome it works as below:
![image](https://github.com/sympa-community/sympa/assets/5743546/019e55c5-2ce1-4848-a9b3-20535ca51662)

**Comment by rm1-uiuc** [comment]
The default focus indicator should not be relied upon, as the default indicator in some browsers is not easily perceivable by some users with visual impairments.

**Comment by ikedas** [comment]
> The default focus indicator should not be relied upon, as the default indicator in some browsers is not easily perceivable by some users with visual impairments.

Could you please present an easily recognizable one across browsers?  I rather think that the arbitrary focus indicator designs that, instead of the default one, vary from application to application will confuse users.

**Comment by ikedas** [comment]
If no good alternative can be found soon, we'd be better to adopt the default one for the time being.
